src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -19865,11 +19839,7 @@ class CL0<T>
 }
 ", NonNullTypesTrue, NonNullTypesAttributesDefinition }, parseOptions: TestOptions.Regular8);
 
-            c.VerifyDiagnostics(
-                // (22,22): warning CS8625: Cannot convert null literal to non-nullable reference or unconstrained type parameter.
```

Comment:
>// (22,22): warning CS8625: Cannot convert null literal to non-nullable reference or unconstrained type parameter. [](start = 16, length = 114)

This warning is correct. For instance if `T` is `string`, this is a warning for `return (string)null;`.